upload/upload.go

```diff
@@ -138,7 +138,7 @@ func uploadFiles(files, outFiles []string, targetDir string, config *helpers.Con
 		}
 		fileExists, err := helpers.ListFiles(*config, listPrefix)
 		if err != nil {
-			log.Error("Couldn't get the file list ", err)
```

Comment:
It would also be a good idea to change the order of doing things here below.
Now we call `ListFiles` first and then check if the `forceOverwrite` flag is set.
We could move the check before calling `ListFiles`. This way if the flag is set, we avoid making that call.
If you think this should be another issue just, disregard the comment.